src/context/forms.js

```diff
@@ -0,0 +1,20 @@
+import { createContext, useState } from 'react';
+
+const FormContext = createContext();
+
+const FormProvider = ({ children }) => {
```

Comment:
not sure why you have gone this approach, to use a Context for form data, typically we'd just `useState` for forms. I will approve it for this time, but otherwise I'd ask to refactor and use the `useState` approach.